qubes/api/internal.py

```diff
@@ -361,3 +361,65 @@ async def suspend_post(self):
                         qubes.config.suspend_timeout,
                         "qubes.SuspendPostAll",
                     )
+
+    @qubes.api.method(
+        "internal.dispvm.preload.Create", scope="local", write=True
+    )
+    async def dispvm_preload_create(self):
+        """
+        Method called to create a preloaded DispVM.
+
+        :return:
+        """
+        dispvm_template = self.arg
```

Comment:
`self.dest`?

Reply:
But also, who will call it? If something from inside qubesd, then it doesn't need to be an "API" method, just a function somewhere, probably next to the dispvm class.